examples/nuxt-openai/package.json

```
@@ -9,18 +9,20 @@
     "postinstall": "nuxt prepare"
   },
   "devDependencies": {
-    "@nuxt/devtools": "0.6.4",
-    "@nuxtjs/tailwindcss": "^6.7.2",
-    "@types/node": "^18",
+    "@nuxt/devtools": "0.6.7",
+    "@nuxtjs/tailwindcss": "^6.8.0",
+    "@types/node": "^20.4.1",
     "@vue/reactivity": "^3.3.4",
     "@vue/runtime-core": "^3.3.4",
     "@vue/runtime-dom": "^3.3.4",
     "@vue/shared": "^3.3.4",
-    "ai": "2.1.15",
-    "nuxt": "^3.5.2",
-    "openai-edge": "^0.5.1",
+    "ai": "2.1.17",
+    "h3": "^1.7.1",
+    "nuxt": "^3.6.2",
```

(for testing - lockfile needs reneal)
```suggestion
    "nuxt": "npm:nuxt3@latest",
```